src/main/java/com/PerkController.java

```diff
@@ -1,14 +1,35 @@
 package com;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.validation.BindingResult;
-import org.springframework.web.bind.annotation.ModelAttribute;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.*;
 
 @Controller
 public class PerkController {
 
+    @Autowired
     private PerkService perkService;
+
```

Comment:
If you are going to use the PerkController like this it needs to be a RestController.

Reply:
Done.